internal/pkg/v2/bootstrap/container/config.go

```diff
@@ -0,0 +1,20 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package container
+
+import (
+	"github.com/edgexfoundry/edgex-go/internal/core/data/config"
```

Comment:
This is core data specific, but in the edgex-go common code. This needs to be changed to not uses core data's specific ConfigurationStruct. There is an interface in go-mod-bootstrap/config/configuration designed to handle the config structs generically.

Reply:
Move to `/internal/core/data/v2 `.